### ISSUE TITLE
After installing obs-websocket, obs crashes on launch

### ISSUE THREAD

**Issue by MrOaiki** [Palakis/obs-websocket#195]
##### Issue type
Bug report?

##### Description
OBS works. Until I install obs-websocket for Mac OS. Then I get a "Click Reopen to open the application again. Click Report to see more detailed information and send a report to Apple." on launch. If I then choose to reload the software, it says "Failed to find locale/en-US.ini" followed by Failed to load locale.

##### Steps to reproduce and other useful info
Install OBS. Install obs-websocket. Crash.

##### Technical information
- Mac OS 10.13.3 (17D102
- OBS 21.0.3

**Comment by JustinLove** [comment]
Same.

Mac OS 10.13.3
OBS 21.0.3

**Comment by staude** [comment]
Same here

Mac OS 10.13.3
OBS 21.0.3

**Comment by staude** [comment]
i downgrade OBS to 20.1, same effect.

**Comment by stkjj** [comment]
seems to be a 10.13. prob. Having macOS 10.12.6 installed, OBS starts up with websocket included without any probs

**Comment by staude** [comment]
After downgrade OBS to 20.1 and reinstall the 21.0.3, now the websocket works on Mac OS 10.13.3

**Comment by dx9s** [comment]
In an attempt to narrow down culprits and seeing memory leaks in the console/log output, obs-websocket is one such culprit:

I've tested (on Linux) and OBS appears to be experience memory leaks (1) with obs-websocket.so installed: 21.0.1 (PPA) and obs-websocket (source, 7fdace9b79759ef9d558b5d6f00b1677f2338612 ).. amongst other combinations..

obs-websocket (recent builds or source) cause 1 memory leak on it's own.

obs-ndi (recent builds and source) cause 1 or more memory leaks on it's own..

obs-websocket + obs-ndi installed causes memory leaks as well.

(output from JUST obs-websocket installed)
```
info:      ┃ ┣obs_init_module(obs-websocket.so): 1.018 ms
[...]
info: =================================================
info: Number of memory leaks: 1
```

**Comment by wherget** [comment]
I've just been through this as well. There are some library incompatibility and loading issues with OBS upgarding Qt to 5.10 and some intricacies on the build side of OBS-Websockets.
On a more positive note, I have it working on my end, thanks to in large parts previous work already done in #207. I'll try and figure out the best way to go about getting a working osx build landed soon.

**Comment by ErikBorra** [comment]
I installed the latest osx pkg (obs-websocket 4.3.1) and obs 21.1.1 would crash on launch afterwards. Reinstalling obs 21.1.1 solved the problem and obs-websocket is now installed. I'm on osx 10.13.4

**Comment by dtcooper** [comment]
Confirmed re-installing 21.1.1 over websockets fixes the problem on macOS 10.13.5

**Comment by Palakis** [comment]
Should be fixed in 4.5.0.
